### DISCUSSION TITLE
I cannot find an option (has gone?)

### DISCUSSION THREAD

**Discussion by luckkmaxx** [libre-tube/LibreTube#5214] [Q&A] [answered]
Below the player you can see related content and comments.
There was an option to set related content in a row above the comments, but I cant find it again.

**Comment by M00NJ** [comment, marked as answer]
Yes, I think it has been removed in v0.20.0.
See #5031.

**Comment by luckkmaxx** [comment]
So horizontal way is the only option. No more vertical related content with bigger thumbnails.

**Comment by M00NJ** [comment]
> So horizontal way is the only option.

Yes. 😐
